fontes/avaliador-sintatico/dialetos/avaliador-sintatico-pitugues.ts

```diff
@@ -1756,4 +1757,39 @@ export class AvaliadorSintaticoPitugues
             erros: this.erros,
         } as RetornoAvaliadorSintatico<Declaracao>;
     }
+  analizarTextoDentroDeClasses(): string {
```

Comment:
```suggestion
  analisarTextoDeDocumentacao(): string {
```

A ideia é reusar o método quando possível. Por ora, não vejo motivo de especializá-lo para classe ou para função.